documentation/asciidoc/computers/compute-module/cmio-camera.adoc

```diff
@@ -1,118 +1,331 @@
-== Attaching a Raspberry Pi Camera Module
+== Attach a Raspberry Pi Camera Module
 
-[NOTE]
-====
-These instructions are intended for advanced users, if anything is unclear please use the https://forums.raspberrypi.com/viewforum.php?f=43[Raspberry Pi Camera forums] for technical help.
+The Compute Module has two CSI-2 camera interfaces: CAM1 and CAM0. This section explains how to connect one or two Raspberry Pi Cameras to a Compute Module using the CAM1 and CAM0 interfaces with a Compute Module I/O Board.
 
-Unless explicitly stated otherwise, these instructions will work identically on both the Compute Module 1 and Compute Module 3, attached to a Compute Module IO Board. Compute Module 4 is slightly different, so please refer to the appropriate section.
-====
+IMPORTANT: Camera modules are not hot-pluggable. *Always* power down your board before connecting or disconnecting a camera module.
 
-The Compute Module has two CSI-2 camera interfaces. CAM0 has two CSI-2 data lanes, whilst CAM1 has four data lanes. The Compute Module IO board exposes both of these interfaces. Note that the standard Raspberry Pi devices use CAM1, but only expose two data lanes.
+=== Update your system
 
-Please note that the camera modules are *not* designed to be hot pluggable. They should always be connected or disconnected with the power off.
-
-=== Updating your System
-
-The camera software is under constant development. Please ensure your system is up to date prior to using these instructions.
+Before configuring a camera, ensure your system runs the latest available software:
 
 ----
 sudo apt update
 sudo apt full-upgrade
 ----
 
-=== Crypto Chip
-
-When using the Compute Module to drive cameras, it is NOT necessary to incorporate the crypto chip used on the Raspberry Pi--designed camera boards when attaching the OM5647, IMX219 or HQ Camera Modules directly to the Compute Module carrier board. The Raspberry Pi firmware will automatically detect the Compute Module and allow communications with the Camera Module to proceed without the crypto chip being present.
-
-=== Quickstart Guide
+=== Connect one camera
 
-To connect a single camera:
+To connect a single camera to a Compute Module, complete the following steps:
 
 . Power the Compute Module down.
-. Connect the RPI-CAMERA board and Camera Module to the CAM1 port. As an alternative, the Raspberry Pi Zero camera cable can be used.
+. Connect the Camera Module to the CAM1 port using the RPI-CAMERA board or the Raspberry Pi Zero camera cable.
 +
 image::images/CMIO-Cam-Adapter.jpg[Connecting the adapter board]
 
-. (CM1 & CM3 only) Connect GPIO pins together as shown below.
+. _(CM1, CM3, CM3+, and CM4S only)_: Connect the following GPIO pins with jumper cables:
```

Comment:
I debated this quite a bit!

I'm torn on it because the board name doesn't correspond perfectly with the supported CMs -- specifically, CM4S isn't in the IO Board name even though it's supported by that board. I know it's a niche product but I would prefer to avoid confusion for customers.